Jenkinsfile

```diff
@@ -123,7 +123,7 @@ pipeline {
             steps {
                 deleteDir()
                 unstash 'MathSetup'
-                sh "echo CC=${env.CXX} -Werror > make/local"
+                sh "echo CXX=${env.CXX} -Werror > make/local"
```

Comment:
You also want to set the `BOOST_PARALLEL_JOBS` option to speed up the MPI build as you are no longer setting this variable's default using the `PARALLEL` variables as I have done before.

Reply:
Thanks. Updated.